crypto/asn1/ameth_lib.c

```diff
@@ -282,7 +285,17 @@ void EVP_PKEY_asn1_copy(EVP_PKEY_ASN1_METHOD *dst,
     dst->siginf_set = src->siginf_set;
 
     dst->pkey_check = src->pkey_check;
+    dst->pkey_public_check = src->pkey_public_check;
+    dst->pkey_param_check = src->pkey_param_check;
+
+    dst->set_priv_key = src->set_priv_key;
+    dst->set_pub_key = src->set_pub_key;
+    dst->get_priv_key = src->get_priv_key;
+    dst->get_pub_key = src->get_pub_key;
 
+    dst->dirty_cnt = src->dirty_cnt;
+    dst->export_to = src->export_to;
+    dst->import_from = src->import_from;
```

Comment:
There is a slightly neater fix for this in #11494

Reply:
Nice. I've updated that patch to follow that pattern for EVP_PKEY_meth_copy. It also makes the tests I added pretty redundant so I took those out.